Puzzlepart/did
# Increased Project and Customer keys max length to 12

### Your checklist for this pull request
- [X] Make sure [CHANGELOG.md](https://github.com/Puzzlepart/did/blob/dev/CHANGELOG.md) is update if applicable

### Review checklist

### Description
Please describe your pull request.

### How to test
1. Check out locally with [gh](https://github.com/cli/cli)
2. Create a new customer with key length 12
3. Create a new project with key length 12

### Related issues
Closes #719

### Related PRs
#702
